toolchains/gcc/defs.bzl

```diff
@@ -163,32 +177,34 @@ def host_os() -> str:
 def _shell_quote(xs):
     return cmd_args(xs, quote = "shell")
 
-def _gcc_arm_none_eabi_toolchain_impl(ctx: AnalysisContext) -> list[Provider]:
-    bin = cmd_args(ctx.attrs.distribution[RunInfo], format = "{}/bin")
-
-    platform_name = "arm32"
+def _gcc_toolchain_impl(ctx: AnalysisContext) -> list[Provider]:
+    target_arch = ctx.attrs.distribution[GccDistributionInfo].target_arch
+    target_os = ctx.attrs.distribution[GccDistributionInfo].target_os
+    target_os = "eabi" if target_os == "bare-metal" else target_os
```

Comment:
`eabi` is not the os, and I think even with the os (i.e. if this was for os `linux`) you'd want to specify the eabi?

In releases.bzl you added `x86_64-linux-aarch64-linux-gnu` and `x86_64-linux-arm-linux-gnueabihf` which have eabi `gnu` and `gnueabihf`, respectively

Reply:
changed to none